<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMScriptForm</class>
 <widget class="QWidget" name="SMScriptForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 1px solid grey;
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0.523, stop:0 rgba(187, 187, 187, 255), stop:1 rgba(230, 230, 230, 255));
}

QPushButton {
	border-radius: 5px;
	background-color: rgba(255, 255, 255, 0);
}

QPushButton::hover
{
	border: 1px solid;
	border-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0.523, stop:0 rgba(187, 187, 187, 255), stop:1 rgba(230, 230, 230, 255));
	border-radius: 2px;
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0.523, stop:0 rgba(187, 187, 187, 255), stop:1 rgba(230, 230, 230, 255));
}

QPushButton::pressed
{
 border: 1px solid;
	border-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:1 rgba(174, 174, 174, 255));
 border-radius: 3px;
 background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:1 rgba(174, 174, 174, 255));
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nombreEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Path de ejecuci√≥n:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="pathEdit"/>
          </item>
          <item>
           <widget class="QToolButton" name="pathButton">
            <property name="toolTip">
             <string>Buscar archivo</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Argumentos:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="argumentosEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ejecutarButton">
          <property name="toolTip">
           <string>Ejecutar script</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="recursos.qrc">
            <normaloff>:/iconos/runIcono</normaloff>:/iconos/runIcono</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="toolTip">
           <string>Borrar script</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="recursos.qrc">
            <normaloff>:/iconos/removeIcono</normaloff>:/iconos/removeIcono</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
